crates/project_panel/src/project_panel.rs

```diff
@@ -1738,6 +1740,12 @@ impl ProjectPanel {
                 icon = FileIcons::get_icon(Path::new(&filename), cx);
             }
         }
+
+        let canonical_path = details
+            .canonical_path
+            .clone()
```

Comment:
```suggestion
            .as_ref()
```

Reply:
Oh yeah a bit loose with my clone.

Adjusted in separate commit